src/Tabs.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Tabs = ({
+  tabs,
+  onChange,
+  selectedTab,
+}) => (
+  <>
+    {tabs.map(tab => (
+      <button
+        key={tab.id}
+        type="submit"
+        value={tab.id}
+        onClick={(event) => {
+          const choosenTab = tabs
+            .find(choosentab => choosentab.id === event.target.value);
```

Comment:
```suggestion
            .find(tab => choosentab.id === event.target.value);
```
chosentTab ты уже используешь выше, а тут просто сделал его не camelCase, используй имена переменных которые будет легко отличить друг от друга